searchv2/rules.py

```diff
@@ -132,6 +132,28 @@ def check(self, package: Package) -> CheckResult:
             return CheckResult(score=0, message="Package is deprecated.")
 
 
+class FavoritePackageRule(ScoreRule):
+    """
+    A specific rule that checks if the package is favorite.
+    """
+
+    name: str = "Favorite Package Rule"
+    description: str = "Check if the package is favorite"
+    max_score: int = 20
+    documentation_url: str = f"{DOCS_URL}/rules/favorite-package"
+
+    def check(self, package: Package) -> CheckResult:
+        """
+        Check if the package is favorite.
+        Returns a full score and a success message if the package is not favorite,
+        or a zero score and an error message otherwise.
+        """
+        if package.is_favorite:
+            return CheckResult(score=0, message="Package is favorite.")
```

Comment:
```suggestion
            return CheckResult(score=self.max_score, message="Package is favorite.")
```

Reply:
And done f6421a52ec9320f4843e70c3891389c7ed3eb012